### ISSUE TITLE
Parent classes get the CSS style of the child classes if you add an extra comma

### ISSUE THREAD

**Issue by jesse-deboer** [postcss/postcss-nested#117]
Parent classes get the css from nested classes if you accidentally put a comma at the end. I really don't know if this should be considered to be a bug or an issue .. 😅 

Please see below for an example:

```
.testClass {
  
  .childClass, .secondChildClass, {
  	background:red;  
  }
  
}
```

`.testClass` will get a red background. This does not happen when using SCSS for example.

**Comment by ai** [comment]
It is a bug.

What is the output in this case?

**Comment by jesse-deboer** [comment]
The compiled output is the following:

`.testClass .childClass, .testClass .secondChildClass, .testClass  {
  	background:red;  
  }`

**Comment by ai** [comment]
I think the issue should be somewhere here https://github.com/postcss/postcss-nested/blob/main/index.js#L40-L56

Can I ask you to look? I am working this week on another open-source project.

**Comment by jesse-deboer** [comment]
No problem @ai, I will try to look if I can fix this!

**Comment by jesse-deboer** [comment]
Hi @ai, fixed in PR #118!

**Comment by ai** [comment]
The fix was released in 5.0.5.

Thanks for the report and fix.
